### PR TITLE
IKC- 356 functions groups

### PR DESCRIPTION
**Pull request by pbelke**

### CODE REVIEW

File: kouncil-backend/src/main/java/com/consdata/kouncil/model/admin/Function.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Table(name = "FUNCTION")
+@Getter
+@Setter
+public class Function {
```
Comment (juris-97) [inline kouncil-backend/src/main/java/com/consdata/kouncil/model/admin/Function.java:25]: IMO Function is very common name, maybe it better to name Feature or Functionality?

File: kouncil-frontend/libs/common-auth/src/lib/auth/kouncil-role.ts [hunk 2]
```diff
@@ -1,6 +1,23 @@
 export enum KouncilRole {
```
Comment (juris-97) [inline kouncil-frontend/libs/common-auth/src/lib/auth/kouncil-role.ts:1]: Not sure if its roles anymore. KouncilFeature?

File: kouncil-backend/src/main/java/com/consdata/kouncil/model/admin/Function.java (hunk 1 above)
Comment (pbelke) [reply kouncil-backend/src/main/java/com/consdata/kouncil/model/admin/Function.java]: Called this entity `SystemFunction`

File: kouncil-frontend/libs/common-auth/src/lib/auth/kouncil-role.ts (hunk 2 above)
Comment (pbelke) [reply kouncil-frontend/libs/common-auth/src/lib/auth/kouncil-role.ts]: To keep everything similar between FE and BE called this enum `SystemFunctionName`

**Review by juris-97 — APPROVED** [review]

**Pull request merged**